python/tskit/tables.py

```diff
@@ -657,41 +656,10 @@ def __str__(self):
         return util.unicode_table(rows, header=headers, row_separator=False)
 
     def _repr_html_(self):
-        """
```

Comment:
Maybe we don't want to delete this docstring, as it could be used in the documentation for the table class?

Reply:
Yes, I guess so. I can alway reinstate. It's not used in the docs ATM anyway (although the `_repr_html_` methods on tree sequences and trees *are* listed)

Reply:
Fixed